candles/sync_candles.py

```diff
@@ -178,39 +178,9 @@ class SyncBitfinexCandles(BaseSyncCandles):
 
     DEFAULT_SYNC_DAYS = 90
     API_MAX_RECORDS = 10000
-    API_CALLS_PER_MIN = 100000 if os.getenv("_").endswith("pytest") else 60
+    API_CALLS_PER_MIN = 100000 if os.getenv("_", "").endswith("pytest") else 60
     EXCHANGE = "bitfinex"
-    PERIODS = [
-        "p2",
-        "p3",
-        "p4",
-        "p5",
-        "p6",
-        "p7",
-        "p8",
-        "p9",
-        "p10",
-        "p11",
-        "p12",
-        "p13",
-        "p14",
-        "p15",
-        "p16",
-        "p17",
-        "p18",
-        "p19",
-        "p20",
-        "p21",
-        "p22",
-        "p23",
-        "p24",
-        "p25",
-        "p26",
-        "p27",
-        "p28",
-        "p29",
-        "p30",
-    ]
+    PERIODS = [f"p{n}" for n in range(2, 31)]
```

Comment:
f-strings are my favourite thing about Python 3.6, and are roughly equivalent to `"p{n}".format(**locals())`

Reply:
Nice - I remember reading about these but thought it was some version later than 3.6. Thanks!